### PR TITLE
fix pipeline build issue

### PR DESCRIPTION
**Pull request by wengyanqing**
## Description
Fix build pipeline issue. Use getenv function to avoid dependency on specific plcontainer extension code. 

## Tests
Since it's a compile issue. Existing regression testing is already enough.

### CODE REVIEW

**Comment by wengyanqing** [comment]
@HaozhouWang @gfphoenix78 please help review.

File: src/common/comm_server.c [hunk 1]
```diff
@@ -199,7 +199,7 @@ int start_listener() {
 	if (strcasecmp("true", use_container_network) == 0) {
 		sock = start_listener_inet();
 	} else if (strcasecmp("false", use_container_network) == 0){
-		if ((CurrentBackendType == BACKEND_DOCKER) && (geteuid() != 0 || getuid() != 0)) {
```
Comment (gfphoenix78) [inline src/common/comm_server.c]: Is there any problem to check `CurrentBackendType`? It's wired to add an additional env for this issue.

Comment (wengyanqing) [reply src/common/comm_server.c]: CurrentBackendType is a global variable in GPDB side, could not refer in pyclient/rclient side, thus it happens a compile issue.

Comment (gfphoenix78) [reply src/common/comm_server.c]: Got it. But it's better to set an env in every server process(docker mode or process mode).

**Pull request merged**